plugins/tiddlywiki/highlight/usage.tid

```diff
@@ -17,9 +14,20 @@ Syntax highlighting is triggered when you add language information to code block
 
 ! Adding Themes
 
-You can add themes from highlight.js by copying the CSS to a new tiddler and tagging it with [[$:/tags/Stylesheet]]. The available themes can be found on GitHub:
+The available themes can be found from the following CDNs:
+
+* <a href=`https://www.jsdelivr.com/package/gh/highlightjs/cdn-release?path=build%2Fstyles&version=$(highlightVersion)$` class="tc-tiddlylink-external" target="_blank">jsDelivr</a>
+* <a href=`https://unpkg.com/browse/@highlightjs/cdn-assets@$(highlightVersion)$/styles/` class="tc-tiddlylink-external" target="_blank">unpkg</a>
```

Comment:
For me this link jumps to the wrong URL -- It opens: https://app.unpkg.com/@highlightjs/cdn-assets@11.4.0/files/languages which imo is not what the link intends to open.